### PR TITLE
Select cameras by name

### PR DESCRIPTION
**Pull request by thomaswmorris**
Selecting by index is bad, because the list of available cameras is not static in practice.

### CODE REVIEW

File: ophyd_basler/utils.py [hunk 1]
```diff
@@ -20,6 +20,19 @@ def configure_logger(logger, log_level=logging.DEBUG, handlers=[StreamHandler]):
...
+    fig, ax = plt.subplots(nrows=1, ncols=1)
+
+    for row in range(1):
+        print(f"{row = }")
+        for col in range(1):
+            print(f"  {col = } --> {ax[row][col]}")
+            ax[row][col].imshow(data[row * 1 + col], vmin=data.min(), vmax=data.max())
+
+    if save_path is not None:
+        plt.savefig(save_path)
```
Comment (mrakitin) [inline ophyd_basler/utils.py:33]: Not sure if we need it here. This seems to be a version of `plot_images` with the hard-coded `nrows=1` and `ncols=1`. Probably we can either significantly simplify it with `plot_fluxes = functools.partial(plot_images, nrows=1, ncols=1)` or just remove it and define it in the IPython session if needed.

**Review by mrakitin — COMMENTED** [review]
The part with instantiating the cam objects by the name looks good.
I don't think we should include the `plot_fluxes` function (see below).

**Comment by BriannaRomasky** [comment]
Looks good!

**Review by BriannaRomasky — APPROVED** [review]
Looks good!

**Pull request merged**